src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -118,7 +125,10 @@ def __call__(
                 noise_pred = noise_pred_uncond + guidance_scale * (noise_pred_text - noise_pred_uncond)
 
             # compute the previous noisy sample x_t -> x_t-1
-            latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs)["prev_sample"]
+            if isinstance(self.scheduler, LmsDiscreteScheduler):
+                latents = self.scheduler.step(noise_pred, i, latents, **extra_step_kwargs)["prev_sample"]
```

Comment:
shouldn't we maybe pass the sigma here?

Reply:
Ok we need it anyway - good to leave as is for me!

Reply:
The `i` is needed inside the `step` for other calculations too, and we can't reverse the sigma into `i`, so we'll need it like this until a more full refactor.